### DISCUSSION TITLE
v-date-input with Supabase (dateformat->yyyy-mm-dd)

### DISCUSSION THREAD

**Discussion by Rasenkantenstein** [vuetifyjs/vuetify#20570] [Q&A]
Hi Community,

I am using Supabase as a backend for my NuxtJS app. I have a question regarding best practises with dates and Vuetify3. 
Supabase returns dates in the format YYYY-MM-DD and also needs that format for write it back to the database. The JS date object throws errors. However, the Vuetify v-date-input does not accept strings but only the date object.

I wonder what is the best way to handle dates in the vue-app in this situation. Should I convert all dates coming from supabase so they represent the date object (e.g. in the store)  or should I rather convert all the dates for the date-input ad-hoc and keep the dates in the store in the supabase string format?

Or, is there actually a way to also use a string format for the v-date-input and let vuetify worry about the date formats?

**Comment by angrymarker** [comment]
Not familiar withSupabase. Does it work with `<v-text-field  type="date" ...>` ?
